config/routes.rb

```diff
@@ -3,7 +3,9 @@
   ActiveAdmin.routes(self)
   root to: 'application#index'
   api_version(module: 'api/v1', path: { value: 'api/v1' }) do
-    resources :books, only: [:index, :show]
+    resources :books, only: [:index, :show, :book_info_external] do
+      get :book_info_external, on: :collection
```

Comment:
we usually agroup all endpoints from member/collection, just like this
```
collection do
  get :book_info_external
end
```
so you don't need to use `on: :collection` in every endpoint